EarlGrey/Synchronization/GREYUIWebViewIdlingResource.m

```diff
@@ -137,6 +137,11 @@ - (BOOL)isIdleNow {
     }
   }
 
+  // UIWebViews may be used to display PDFs in addition to HTML based content. Test if there's a PDF
+  // view populated in UIWebView's hierarchy.
+  BOOL webViewIsDisplayingPDF = ([[[_webView valueForKey:@"_internal"] valueForKey:@"pdfHandler"]
```

Comment:
> Do we have to do something with the bool?
Yup, need to return that we are idle.

> Can we add a test case with a PDF to prevent regressions & prove the issue is fixed?
Sure, let me put something together.

Reply:
I wired up a basic test case that shows that opening a PDF will cause the app to never reach an idle state. PTAL.

Reply:
Nice, looks good to me. Someone on the EarlGrey will need to approve before the PR is merged.